src/SDKs/DevSpaces/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag />

```

Comment:
@sanar-microsoft missing meta data.
Can you build the generated SDK as
msbuild build.proj /t:Build /p:Scope=SDKs\DevSpaces